Dockerfile.BUILDS_AND_CDN

```diff
@@ -15,6 +15,7 @@ RUN yarn verify
 # Build project
 ARG environment
 ENV REACT_APP_ROUTER=hash
+ENV PUBLIC_URL="."
```

Comment:
You should add this in all files where `REACT_APP_ROUTER=hash`, right?

Reply:
On the contrary, if you merge it, you will break the production environment.

Reply:
669df940dc56294fc180661f719b82896e043b11 Oh, I understand, you want to merge it and apply the changes to PROD  all in the same step. 

I was understanding that the scope of this PR was only fixing INT.

Reply:
This file only has effect in cdn build, for releases we use `DockerFile.RELEASES`. In all the rest of the builds, url will be static, since we don't use '.' in homepage. This can be with or without router change. 
If I merge this PR the changes over config files will not be applied because on service creation we are using secrets. It should be replaced for current secret in production and QA.